db/users/update.go

```diff
@@ -0,0 +1,70 @@
+package users
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/go-scalingo/v6"
+	"github.com/Scalingo/go-utils/errors/v2"
+	"github.com/Scalingo/gopassword"
+)
+
+func UpdateUser(ctx context.Context, app, addonUUID, username string) error {
+	isSupported, err := doesDatabaseHandleUserManagement(ctx, app, addonUUID)
+	if err != nil {
+		return errors.Wrap(ctx, err, "get user management information")
+	}
+
+	if !isSupported {
+		io.Error(ErrDatabaseNotSupportUserManagement)
+		return nil
+	}
+
+	if usernameValidation, ok := isUsernameValid(username); !ok {
+		io.Error(usernameValidation)
+		return nil
+	}
+
+	password, confirmedPassword, err := askForPassword(ctx)
+	if err != nil {
+		return errors.Wrap(ctx, err, "ask for password")
+	}
+
+	passwordValidation, ok := isPasswordValid(password, confirmedPassword)
+	if !ok && password != "" {
+		io.Error(passwordValidation)
+		return nil
+	}
+
+	isPasswordGenerated := false
+	if password == "" {
+		isPasswordGenerated = true
```

Comment:
suggestion: display a message that password will be generated because left empty or max retry attempts reached

Reply:
It's already handled by `askForPassword` or you're looking for something else?

Reply:
I'm not really sure what you means.

But I'm not sure we want to change the behavior as it is what the API expects actually, empty password and confirmation to make it generate one: https://developers.scalingo.com/databases/users

I've updated the description of this PR with an example of generated password. It is more clear and/or ok for you?

Reply:
Yup, sorry misread your code :slightly_frowning_face:

Reply:
No problem